src/names.c

```diff
@@ -17,6 +17,12 @@ r_obj* vec_as_names(r_obj* names, const struct name_repair_opts* opts) {
   case NAME_REPAIR_unique: return vec_as_unique_names(names, opts->quiet);
   case NAME_REPAIR_universal: return vec_as_universal_names(names, opts->quiet);
   case NAME_REPAIR_check_unique: return check_unique_names(names, opts);
+  // I have to handle these elements of the enum to please the compiler, but the
```

Comment:
I think it's ok to give precedence to the enum setting. And it's also ok to support options that are not advertised.